dask/array/tests/test_cupy.py

```diff
@@ -956,3 +956,88 @@ def test_sparse_dot(sp_format):
 
     assert cupyx.scipy.sparse.isspmatrix(da_z)
     assert_eq(z, da_z.todense())
+
+
+@pytest.mark.skipif(np.__version__ < "1.20", reason="NEP-35 is not available")
```

Comment:
Nitpick: could you use `_numpy_120` here (and in the other tests) instead of `np.__version__` directly?

https://github.com/dask/dask/blob/2c2e837674895cafdb0612be81250ef2657d947e/dask/array/numpy_compat.py#L11

Using these compatibility variables makes it easier to search through the codebase for tests that depend on specific NumPy version

Reply:
```suggestion
@pytest.mark.skipif(_numpy_120, reason="NEP-35 is not available")
```

Reply:
I think we want `not _numpy_120` here to make sure we skip if NumPy is less than 1.20

Reply:
Right sorry about that. Fixed

Reply:
Thanks @jrbourbeau , I wasn't aware of that constant. I updated the tests with that now.